lib/index.ts

```diff
@@ -382,7 +382,6 @@ export const traceHintProps = z.object({
   order: z.number().optional(),
   offset: route_hint_point.optional(),
   offsets: z.array(route_hint_point).optional(),
-  traceWidth: z.number().optional(),
```

Comment:
hmm intentional? I think this should also be supported, you can do either, both options have different precedence

Reply:
Yeah, I am following the syntax where the traceWidth is added inside of the route. And then the default takes precedence

Reply:
so then `traceWidth` should exist here too right, both options should be supported.

Reply:
also `traceWidth` should use the `length` zod type, that way it can be defined like "0.1mm"

Reply:
Okay, we can add the support for this as well. Will make the update for the soup as well

Reply:
@imrishabh18 FWIW it's okay for the soup to pre-compute the width of each point on the route. The way to think about this is "props" is ergonomic for users and "soup" is ergonomic for _renderers_, we don't want the renderer to have to contain the logic to figure out what the width is at each point.